rasplayer.js

```diff
@@ -127,7 +138,8 @@ function sendStatus(omx, socket) {
 
 function manualPlay(omx) {
   // Play the video
-  omx.play(omx.response.playlist[omx.nextVideo], {'-o': omx.response.audioOutput, '--display': 5});
+  player = omx.create(omx.response.playlist[omx.nextVideo], {'-o': omx.response.audioOutput, '--display': 5, '-b': true});
```

Comment:
As part of my update code, I changed the display logic to blank out any borders around the video (I didn't care to have the background visible behind the video).  An alternative here would be to add another configuration checkbox to the playlist UI to control this display element.

Reply:
I like this solution, the intent was always to use this as a kiosk type system, so the background being visible is not desirable